packages/amplify-category-predictions/provider-utils/awscloudformation/assets/regionMapping.js

```diff
@@ -0,0 +1,131 @@
+
+const Translate = {
+  'us-east-1': 'us-east-1',
+  'us-east-2': 'us-east-1',
+  'us-west-2': 'us-west-2',
+  'eu-west-1': 'eu-west-1',
+  'eu-west-2': 'eu-west-1',
+  'eu-central-1': 'eu-central-1',
+  'ap-northeast-1': 'ap-northeast-1',
+  'ap-northeast-2': 'ap-northeast-2',
+  'ap-southeast-1': 'ap-southeast-1',
+  'ap-southeast-2': 'ap-southeast-1',
+  'ap-south-1': 'ap-south-1',
+};
+
+
+const Polly = {
+  'us-east-1': 'us-east-1',
+  'us-east-2': 'us-east-2',
+  'us-west-2': 'us-west-2',
+  'eu-west-1': 'eu-west-1',
+  'eu-west-2': 'eu-west-2',
+  'eu-central-1': 'eu-central-1',
+  'ap-northeast-1': 'ap-northeast-1',
+  'ap-northeast-2': 'ap-northeast-2',
+  'ap-southeast-1': 'ap-southeast-1',
+  'ap-southeast-2': 'ap-southeast-2',
+  'ap-south-1': 'ap-south-1',
+};
+
+const Transcribe = {
+  'us-east-1': 'us-east-1',
+  'us-east-2': 'us-east-2',
+  'us-west-2': 'us-west-2',
+  'eu-west-1': 'eu-west-1',
+  'eu-west-2': 'eu-west-2',
+  'eu-central-1': 'eu-central-1',
+  'ap-northeast-1': 'ap-northeast-2',
+  'ap-northeast-2': 'ap-northeast-2',
+  'ap-southeast-1': 'ap-southeast-1',
+  'ap-southeast-2': 'ap-southeast-2',
+  'ap-south-1': 'ap-south-1',
+};
+
+const Rekognition = {
+  'us-east-1': 'us-east-1',
+  'us-east-2': 'us-east-2',
+  'us-west-2': 'us-west-2',
+  'eu-west-1': 'eu-west-1',
+  'eu-west-2': 'eu-west-2',
+  'eu-central-1': 'eu-central-1',
+  'ap-northeast-1': 'ap-northeast-1',
+  'ap-northeast-2': 'ap-northeast-2',
+  'ap-southeast-1': 'ap-southeast-1',
+  'ap-southeast-2': 'ap-southeast-2',
+  'ap-south-1': 'ap-south-1',
+};
+
+const RekognitionAndTextract = {
+  'us-east-1': 'us-east-1',
+  'us-east-2': 'us-east-2',
+  'us-west-2': 'us-west-2',
+  'eu-west-1': 'eu-west-1',
+  'eu-west-2': 'us-east-1',
+  'eu-central-1': 'us-east-1',
+  'ap-northeast-1': 'us-east-1',
+  'ap-northeast-2': 'us-east-1',
+  'ap-southeast-1': 'us-east-1',
+  'ap-southeast-2': 'us-east-1',
+  'ap-south-1': 'us-east-1',
```

Comment:
Can we possibly add a warning message letting them know we are defaulting back to `us-east-1`